### DISCUSSION TITLE
[Lazer] Arrow keys not working in the Input section in settings

### DISCUSSION THREAD

**Discussion by gomi-bear** [ppy/osu#26175] [Q&A] [answered]
### Problem details:
I can't use the arrow keys in the 'Input' section of the settings menu? They work everywhere else, but If you try to adjust specifically the sensitivity or the deadzone they won't work.

I have ZERO clue what could be causing it. It would be one thing if the arrow keys just didn't work, but the fact they work everywhere except the Input section makes me question if it is something on my end.

this problem started 2 updates ago, in Lazer 2023.1223.0
The most recent updated didn't fix it, so I mentioned it under the changelog
and according to Peppy the arrow keys works just fine on their end.

### Video of problem:
You can't tell in the video but I am spamming the right and left arrow keys on the sensitivity and deadzone slider.
https://www.youtube.com/watch?v=aO1UIkA2OeY

**Comment by peppy** [comment]
This is very weird. The only guess I have is that the search textbox is taking more focus than before (a dropdown somewhere in a weird state maybe) cc @frenzibyte might have an idea.

**Reply by gomi-bear** [reply]
It looks like there is something else that is taking more of a focus. I found out I can get the arrow keys to work if I search for either the **sensitivity** or **deadzone**. 

I went letter by letter and after reaching **‘sen’** the sensitivity started to work. It also worked if I removed the “s” and just typed **“en”**.

When I tried **“de”** deadzone started to work. 

When I searched “Input” or “cursor” it didn't work.

https://www.youtube.com/watch?v=LH3jBBKVZ4I

**Reply by frenzibyte** [reply]
I can reproduce this by having the confine dropdown disabled (by switching to full screen mode).

**Comment by frenzibyte** [comment, marked as answer]
Tracking in https://github.com/ppy/osu/issues/26211
